cdk/domino_cdk/config/install.py

```diff
@@ -27,9 +27,17 @@ class Install:
     registry_password: str
     overrides: dict
     istio_compatible: bool
+    enabled: bool = True
```

Comment:
For the install section specifically, saying `enabled` feels a bit weird since this section doesn't actually do anything, it's just informational (ie it generates an agent config template as an output).

I wonder if it would make more sense to make this section entirely optional, so you can just load a config without an install block or perhaps one that's explicitly blank or set to null? Or perhaps that should just be the pattern overall—allow certain blocks to be null? Probably prevents a lot of irrelevant stuff from being in the config and validated too.

The config generator util.py uses could have args that prevent those blocks from being generated in the first place, and then in platform-apps the args sent to the config generator in the manifest could be used.

Just a suggestion, but within the pattern we've used I might either rename this variable to something that indicates it's not generating the config, or just omit it and infer from what modules were disabled that we don't need to generate the agent template.